app/jmeter/jira.jmx

```diff
@@ -1124,14 +1124,14 @@ JMeterUtils.setProperty(&quot;c_AtlToken&quot; + user_counter, atl_token)
             </JSR223PreProcessor>
             <hashTree/>
           </hashTree>
-          <ThroughputController guiclass="ThroughputControllerGui" testclass="ThroughputController" testname="create issue" enabled="true">
+          <ThroughputController guiclass="ThroughputControllerGui" testclass="ThroughputController" testname="create_issue" enabled="true">
             <intProp name="ThroughputController.style">1</intProp>
             <boolProp name="ThroughputController.perThread">false</boolProp>
             <intProp name="ThroughputController.maxThroughput">1</intProp>
             <stringProp name="ThroughputController.percentThroughput">${perc_create_issue}</stringProp>
           </ThroughputController>
           <hashTree>
-            <TransactionController guiclass="TransactionControllerGui" testclass="TransactionController" testname="jmeter_open_quick_create" enabled="true">
+            <TransactionController guiclass="TransactionControllerGui" testclass="TransactionController" testname="jmeter_create_issue:open_quick_create" enabled="true">
```

Comment:
Where are other jmeter_create_issue subactions?

Reply:
![image](https://user-images.githubusercontent.com/30836532/102895223-028f4580-446d-11eb-9ddd-652f084790a9.png)
I think in that case we don't need like that "jmeter_create_issue:create_issue". How do you think?

Reply:
see corresponding selenium action as example

Reply:
We have the same for Selenium

Reply:
<img width="846" alt="Screenshot 2020-12-22 at 16 04 46" src="https://user-images.githubusercontent.com/11671213/102896797-a11ca600-446f-11eb-966f-37b3dc345e0a.png">